experimental/examples/imageviewer/shared/src/desktopMain/kotlin/example/imageviewer/DesktopImageStorage.kt

```diff
@@ -0,0 +1,65 @@
+package example.imageviewer
+
+import androidx.compose.runtime.snapshots.SnapshotStateList
+import androidx.compose.ui.graphics.ImageBitmap
+import androidx.compose.ui.graphics.toAwtImage
+import androidx.compose.ui.graphics.toComposeImageBitmap
+import example.imageviewer.filter.scaleBitmapAspectRatio
+import example.imageviewer.model.PictureData
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.launch
+
+private const val maxStorableImageSize = 2000
+private const val storableThumbnailSize = 200
+
+class DesktopImageStorage(
+    private val pictures: SnapshotStateList<PictureData>,
+    private val ioScope: CoroutineScope
+) : ImageStorage {
+    val mapWithBigImages = mutableMapOf<PictureData.Camera, ImageBitmap>()
+    val mapWithThumbnails = mutableMapOf<PictureData.Camera, ImageBitmap>()
```

Comment:
Can we share this duplicated code?

Reply:
Also on Android, this map will be removed because we will store data on disk storage